framework/src/node/state_machine/types.ts

```diff
@@ -28,7 +28,7 @@ export interface ImmutableSubStore {
 export interface SubStore extends ImmutableSubStore {
 	del(key: Buffer): Promise<void>;
 	set(key: Buffer, value: Buffer): Promise<void>;
-	setWithSchema(key: Buffer, value: Record<string, unknown>, schema: Schema): Promise<void>;
+	setWithSchema(key: Buffer, value: object, schema: Schema): Promise<void>;
```

Comment:
why remove `Record<string, unknown>`?

Reply:
`Record<string, unknown>` is changed to `object` because we want to type the `validatorAccount` as `ValidatorKeys` and then be able to set it using `validatorsSubStore.setWithSchema` which is not possible with `Record<string, unknown>`

```
		const validatorAccount = await validatorsSubStore.getWithSchema<ValidatorKeys>(
			validatorAddress,
			validatorAccountSchema,
		);
		validatorAccount.generatorKey = generatorKey;
		await validatorsSubStore.setWithSchema(
			validatorAddress,
			validatorAccount,
			validatorAccountSchema,
		);
```